### DISCUSSION TITLE
building module quarkus-grpc-stubs failed

### DISCUSSION THREAD

**Discussion by israelstmz** [quarkusio/quarkus#22573] [Q&A]
Hi there,
I am building the quarkus project for the 1st time and getting the following failure on "quarkus-grpc-stubs":

> [INFO] Scanning for projects...
[INFO] ------------------------------------------------------------------------
[INFO] Detecting the operating system and CPU architecture
[INFO] ------------------------------------------------------------------------
[INFO] os.detected.name: windows
[INFO] os.detected.arch: x86_64
[INFO] os.detected.bitness: 64
[INFO] os.detected.version: 10.0
[INFO] os.detected.version.major: 10
[INFO] os.detected.version.minor: 0
[INFO] os.detected.classifier: windows-x86_64
[INFO] 
[INFO] -------------------< io.quarkus:quarkus-grpc-stubs >--------------------
[INFO] Building Quarkus - gRPC - Stubs for health and reflection 999-SNAPSHOT
[INFO] --------------------------------[ jar ]---------------------------------
[INFO] 
[INFO] --- maven-enforcer-plugin:3.0.0-M3:enforce (enforce) @ quarkus-grpc-stubs ---
[INFO] 
[INFO] --- maven-enforcer-plugin:3.0.0-M3:enforce (enforce-test-deps-scope) @ quarkus-grpc-stubs ---
[INFO] 
[INFO] --- maven-enforcer-plugin:3.0.0-M3:enforce (enforce-test-deps-junit-scope) @ quarkus-grpc-stubs ---
[INFO] 
[INFO] --- maven-enforcer-plugin:3.0.0-M3:enforce (enforce-java-version) @ quarkus-grpc-stubs ---
[INFO] 
[INFO] --- maven-enforcer-plugin:3.0.0-M3:enforce (enforce-maven-version) @ quarkus-grpc-stubs ---
[INFO] 
[INFO] --- maven-enforcer-plugin:3.0.0-M3:enforce (enforce-no-runtime-deps) @ quarkus-grpc-stubs ---
[INFO] 
[INFO] --- buildnumber-maven-plugin:1.4:create (get-scm-revision) @ quarkus-grpc-stubs ---
[INFO] Executing: cmd.exe /X /C "git rev-parse --verify HEAD"
[INFO] Working directory: D:\IdeaProjects\quarkus\extensions\grpc\stubs
[INFO] Storing buildNumber: 5c120b1e5c8e4c4bd587581137edd2762472b52f at timestamp: 1641062261694
[INFO] Storing buildScmBranch: main
[INFO] 
[INFO] --- protobuf-maven-plugin:0.6.1:compile (compile) @ quarkus-grpc-stubs ---
[INFO] Building protoc plugin: quarkus-grpc-protoc-plugin
[INFO] Compiling 2 proto file(s) to D:\IdeaProjects\quarkus\extensions\grpc\stubs\target\generated-sources\protobuf\java
[ERROR] PROTOC FAILED: --quarkus-grpc-protoc-plugin_out: protoc-gen-quarkus-grpc-protoc-plugin: Plugin failed with status code 4294967295.
>
> [ERROR] D:\IdeaProjects\quarkus\extensions\grpc\stubs\src\main\proto\health\v1\health.proto [0:0]: --quarkus-grpc-protoc-plugin_out: protoc-gen-quarkus-grpc-protoc-plugin: Plugin failed with status code 4294967295.
>
> [ERROR] D:\IdeaProjects\quarkus\extensions\grpc\stubs\src\main\proto\reflection\v1alpha\reflection.proto [0:0]: --quarkus-grpc-protoc-plugin_out: protoc-gen-quarkus-grpc-protoc-plugin: Plugin failed with status code 4294967295.

any idea how to resolve this? or what it even means?

**windows 10 pro**
**maven 3.8.4**

[a comment by an automated account is not shown]

**Comment by michalszynkiewicz** [comment]
What command do you use exactly?

**Reply by israelstmz** [reply]
`.\mvnw.cmd install -Dquickly -Dmaven.wagon.http.ssl.insecure=true -Dmaven.wagon.http.ssl.allowall=true -Dmaven.wagon.http.ssl.ignore.validity.dates=true`

Can it be related to this issue: [https://github.com/quarkusio/quarkus/issues/10214](url)?

**Reply by michalszynkiewicz** [reply]
Are you building `main` or `master` branch? I tried master by accident when checking this and run in all kinds of trouble but `main`, which is the current development branch, works fine for me (with cleaned up m2)

**Reply by israelstmz** [reply]
I am building main (forked).
Did you build it on windows?

**Reply by michalszynkiewicz** [reply]
no, I only have linux machines... Why do you need -Dmaven.wagon.http.ssl.insecure... ?

**Reply by israelstmz** [reply]
I am behind a proxy. this is only for maven imports to succeed.
I am suspecting windows... because when I try on linux I also succeed...

**Reply by michalszynkiewicz** [reply]
It's tough to help not having an environment that could reproduce the problem. Please let us know if you'd have any more findings
